scripts/verify-knife

```diff
@@ -0,0 +1,42 @@
+#!/bin/sh
+
+# Verify mmfile $1 (default "set.mm") and discouraged file $2 (default
+# "discouraged") using metamath-knife.
+# Usage:
+# verify-knife [--check-newlines] [source=set.mm] [discouraged_file=discouraged]
+
+check_newlines='false'
+
+if [ "$1" = '--check-newlines' ] ; then
+  check_newlines='true'
+  shift
+fi
+
+source="${1:-set.mm}"
+discouraged_file="${2:-discouraged}"
+
+metamath-knife ${source} --verify \
+                         --verify-parse-stmt \
```

Comment:
`--parse-typesetting` disappeared ?

Reply:
`--verify-markup` already performs the typesetting pass, so `--parse-typesetting` is redundant.

Reply:
> --verify-markup already performs the typesetting pass

@tirix can you add that to the doc/manpage of metamath knife ?  On https://github.com/metamath/metamath-knife I see
```
    -t, --parse-typesetting    Parses typesetting information
...
    -m, --verify-markup        Checks comment markup
```
Should this be
```
    -t, --parse-typesetting    Parses typesetting information
...
    -m, --verify-markup        Checks comment markup and parses typesetting information
```
(Also, it seems more natural to me to use imperative everywhere, that is, "parse" instead of "parses", since this is a command, but that's minor.)

Reply:
I've created [metamath-knife#167](https://github.com/metamath/metamath-knife/pull/167) for this.